traitquestdev/my/profile/edit.php

```diff
@@ -111,19 +111,61 @@
 		<div class="col-lg-12 col-md-12 margin-topbottom-s padding-topbottom-l padding-leftright-s border-top-grey white-bg">
 		<div id="mainContainer">
 			<form id="formEditProfile" class="form" method="post">
+				<div id="columnPhone" class="columnInput">
+					<p class="clear-both padding-top-s">Mobile</p>
+					<input type="text" name="phone" id="phone" class="clear-both col-lg-5 col-md-5 col-sm-12 col-xs-12" placeholder="Mobile Phone No." />
+				</div>
+				<div id="" class="columnInput">
+					<p class="clear-both padding-top-s">D.O.B</p>
+					<input type="text" name="" id="" class="clear-both col-lg-5 col-md-5 col-sm-12 col-xs-12" placeholder="Date of Birth" />
+				</div>
+				<div id="" class="columnInput">
+					<p class="clear-both padding-top-s">Nationality</p>
+					<?php include 'countryselect.php' ?>
+				</div>
+				<div id="" class="columnInput">
+					<p class="clear-both padding-top-s">Race</p>
+					<select class="clear-both col-lg-5 col-md-5 col-sm-12 col-xs-12" name="race">
+						<option value="">Select</option>
```

Comment:
Value to be 'Select a race'. And set this option to be disabled so user cant select it.